File: C_HDOJ/HDOJ2028.c
#include <stdio.h>

int main(){
	int i,n;
	while(~scanf("%d",&n)){
		int num[100];
		int max=-1;
		for(i=0;i<n;i++){
			scanf("%d",&num[i]);
			if(num[i]>max){
				max=num[i];
			}
		}
		for(i=0;i<n;i++){
			if(max%num[i]!=0){
				i=-1;
				max++;
			}
		}
		printf("%d\n",max);
	}
	return 0;
}

File: Java_HDOJ/src/HDOJ2028.java
import java.math.BigInteger;
import java.util.Scanner;

public class HDOJ2028 {
    public static void main(String[] args) {
        Scanner sc=new Scanner(System.in);
        int num;
        while(sc.hasNext()){
            int a=sc.nextInt();
            int b=sc.nextInt();
            int max;
            max=Math.max(a,b);
            while(max%a!=0||max%b!=0){
                max++;
            }
            System.out.println(max);
        }
    }
}
